### PR TITLE
Show incompatible devices when API is missing from trace

### PR DESCRIPTION
**Pull request by hliatis**
Bug: b/173108206

### CODE REVIEW

File: gapis/messages/en-us.stb.md [hunk 1]
```diff
@@ -258,3 +258,7 @@ Device graphics API version ({{device_api_version}}) is different from the one o
 # REPLAY_COMPATIBILITY_INCOMPATIBLE_API
 
 Device does not support the graphics API of the trace.
+
+# REPLAY_COMPATIBILITY_MISSING_API
+
+Trace does not provide a graphics API.
```
Comment (pmuetschard) [inline gapis/messages/en-us.stb.md:264]: "Trace file is incomplete/truncated: it does not contain any traced commands."

File: gapis/replay/devices/devices.go [hunk 2]
```diff
@@ -79,6 +86,7 @@ func ForReplay(ctx context.Context, p *path.Capture) ([]*path.Device, []bool, []
 			}.Bind(ctx)
 			log.D(ctx, "Priority %d", p)
 		}
+
```
Comment (hevrard) [inline gapis/replay/devices/devices.go:89]: Nit: remove this empty line

**Review by hevrard — APPROVED** [review]
Just an empty line diff to remove. Thanks for this!

File: gapis/messages/en-us.stb.md [hunk 3]
```diff
@@ -258,3 +258,7 @@ Device graphics API version ({{device_api_version}}) is different from the one o
 # REPLAY_COMPATIBILITY_INCOMPATIBLE_API
 
 Device does not support the graphics API of the trace.
+
+# REPLAY_COMPATIBILITY_MISSING_API
+
+Trace file is incomplete/truncated: it does not contain any traced commands
```
Comment (pmuetschard) [inline gapis/messages/en-us.stb.md:264]: Add a period at the end of the sentence, please.

**Review by pmuetschard — APPROVED** [review]

**Pull request merged**